library/src/meta/footnote.rs

```diff
@@ -201,7 +298,7 @@ impl Show for FootnoteEntry {
             HElem::new(self.indent(styles).into()).pack(),
             sup,
             HElem::new(number_gap.into()).with_weak(true).pack(),
-            note.body(),
+            note.body_content().unwrap(),
```

Comment:
could this panic if a footnote entry is manually constructed?

Reply:
I don't think so. The entry is only created if the footnote has a body, in `FlowLayouter::handle_footnotes`, thanks to
```rust
if notes[k].is_ref() {
    k += 1;
    continue;
}
```
in `flow.rs`, line 491.